extra-utils/genie/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=genie
+PKGSEC=utils
+PKGDEP="daemonize dbus dotnet-runtime-3.1 dotnet-sdk-3.1 systemd"
+PKGDES="A quick way into a systemd \"bottle\" for WSL"
```

Comment:
This `PKGDES` does not make linguistic sense.

Reply:
Updated